### PR TITLE
CIF-1380 - Product collection component styling

### PR DESCRIPTION
**Pull request by LSantha**
* created v2 of productcollection and searchresults component
 * cleaned up CSS classes, markup and HTL scripts for v2 of productcollection, productlist and searchresults component
 * updated component READMEs

## Related Issue

CIF-1380

## How Has This Been Tested?

Manually. JUnit.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [X] I have added tests to cover my changes and the overall coverage did not decrease.
- [X] All unit tests pass on CircleCi.
- [X] I ran all tests locally and they pass.

### CODE REVIEW

**Review by Buuhuu — APPROVED** [review]
lgtm

File: ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productcollection/v2/productcollection/README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    ELEMENT productcollection__pagination-button
+        MOD productcollection__pagination-button--current
+        MOD productcollection__pagination-button--inactive
+    ELEMENT productcollection__pagination-button-title                
+```
+
+## Information
+* **Vendor**: Adobe
+* **Version**: v1
+* **Compatibility**: AEM as a Cloud Service / AEM 6.4 / 6.5
```
Comment (mhaack) [inline ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productcollection/v2/productcollection/README.md:92]: I suggest to only use AEM as a Cloud Service & 6.5 for all new components, we don't really test with 6.4 any more and other parts are also only supported on AEM as a Cloud Service & 6.5

Comment (LSantha) [reply ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productcollection/v2/productcollection/README.md]: OK, I've fixed it.

**Review by herzog31 — APPROVED** [review]

File: ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productcollection/v2/productcollection/README.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        MOD productcollection__filter-icon--closed    
+    ELEMENT productcollection__filter-items
+    ELEMENT productcollection__filter-item
+    ELEMENT productcollection__filter-title
+    ELEMENT productcollection__items
+    ELEMENT productcollection__item
+        MOD productcollection__item--staged
+    ELEMENT productcollection__item-images
+    ELEMENT productcollection__item-image
+    ELEMENT productcollection__item-image-placeholder
```
Comment (Buuhuu) [inline ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productcollection/v2/productcollection/README.md:69]: Isn't that a `MOD` ? At least is see it used in combination with `productcollection__item-image`

Comment (LSantha) [reply ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/productcollection/v2/productcollection/README.md]: Indeed, it is. I've fixed it.

**Pull request merged**